cms/urls.py

```diff
@@ -1,14 +1,14 @@
 from django.conf import settings
 from django.contrib import admin
 from django.urls import include, path
-
 from wagtail.admin import urls as wagtailadmin_urls
+from wagtail.contrib.sitemaps.views import sitemap
```

Comment:
This looks errant

Reply:
That's referenced on line 15

```
path("sitemap.xml", sitemap),
```

It just got moved from line 7. Jet brains has an "optimise imports" macro that re-orders and sorts imports into groups and alpha order.